### PR TITLE
[Homework #21 ] csr-ssr-ssg-nextjs by Julian Oviedo

### PR DESCRIPTION
**Pull request by JulianOviedo**
Hi Ilya ! here is my hw21. Please, let me know if i need to improve something.

Thank you !!

### CODE REVIEW

File: homeworks/julian-oviedo/21-csr-ssr-ssg-nextjs/avatar-next-app/pages/index.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      const preFetch = dataJson.map((avatarImgs) => avatarImgs.url);
+      return {
+        props: { preFetchedAvatars: preFetch },
+      };
+    });
+};
+
+export default function Home({ preFetchedAvatars }) {
+  const [avatars, setAvatars] = useState(preFetchedAvatars);
+  console.log(avatars);
```
Comment (ilya-ischenko) [inline homeworks/julian-oviedo/21-csr-ssr-ssg-nextjs/avatar-next-app/pages/index.js:29]: Remove, please

**Review by ilya-ischenko — COMMENTED** [review]
Good practice to create "src" folder and move all files related to react app: components, hooks, styles

Also I see warning "Parsing error: Cannot find module 'next/babel'" almost in every .js file, you can try to fix it, but it is on your own

File: homeworks/julian-oviedo/21-csr-ssr-ssg-nextjs/avatar-next-app/pages/index.js (hunk 1 above)
Comment (JulianOviedo) [reply homeworks/julian-oviedo/21-csr-ssr-ssg-nextjs/avatar-next-app/pages/index.js:29]: > Good practice to create "src" folder and move all files related to react app: components, hooks, styles
> 
> Also I see warning "Parsing error: Cannot find module 'next/babel'" almost in every .js file, you can try to fix it, but it is on your own

Hi Ilya. Where do you see the warning ? When i run the server i dont see any error/warning. Could u please show me ?

Comment (ilya-ischenko) [reply homeworks/julian-oviedo/21-csr-ssr-ssg-nextjs/avatar-next-app/pages/index.js]: My bad, sorry, I opened project in wrong directory

**Review by ilya-ischenko — APPROVED** [review]
Well done!
Approved.

**Pull request merged**